### PR TITLE
add gds careers to transition

### PR DESCRIPTION
**Pull request by JonathanHallam**
Trello - https://trello.com/c/pWbH2Z31/750-redirect-gdscareersgovuk-pages-to-the-new-careers-page

This repo is owned by the publishing platform team. Please let us know in #govuk-publishing-platform when you raise any PRs.

### CODE REVIEW

File: data/transition-sites/gds_career.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+site: gds_career
+whitehall_slug: government-digital-service
+homepage: https://www.gov.uk/government/organisations/government-digital-service
+tna_timestamp: 20201010101010 # Stub timestamp - site is not in TNA
+host: www.civil-service-careers.gov.uk
```
Comment (brucebolt) [inline data/transition-sites/gds_career.yml:6]: Looking at the ticket, shouldn't the host be `gdscareers.gov.uk`?

```suggestion
host: gdscareers.gov.uk
```

Comment (JonathanHallam) [reply data/transition-sites/gds_career.yml:6]: docs lead me to believe that it was "the new homepage for the old site" but I'll admit they're not especially clear and I could be reading it wrong 

https://github.com/alphagov/transition-config/blob/main/README.md

Comment (brucebolt) [reply data/transition-sites/gds_career.yml:6]: I thought we configured the old domain here, then set up the new URL in the transition UI?

Comment (brucebolt) [reply data/transition-sites/gds_career.yml:6]: Or set the new URL as the `homepage` value?

Comment (JonathanHallam) [reply data/transition-sites/gds_career.yml:6]: Just checked some of the others, you're 100% right, will amend!

Comment (JonathanHallam) [reply data/transition-sites/gds_career.yml]: done!

**Review by brucebolt — APPROVED** [review]

**Pull request merged**